mssw/Acme-Spinodal-Utils/t/Utils.t

```diff
@@ -0,0 +1,51 @@
+package t::Acme::Spinodal::Utils;
+
+use strict;
+use warnings;
+use v5.12;
+
+use Data::Dumper;
+
+use Test::More;
+use Test::Exception;
+
+BEGIN {
+  use_ok( 'Acme::Spinodal::Utils' ) || print "Bail out!\n";
```

Comment:
Might as well import the function you'll be testing (`Acme::Spinodal::Utils->import( 'sum' )`).